SpectraLogic.EscapePodClient/Runtime/Network.cs

```diff
@@ -60,6 +74,19 @@ public IHttpWebResponse Invoke(RestRequest request)
             }
         }
 
+        private static string BuildQueryParams(Dictionary<string, string> queryParams)
+        {
+            return string.Join(
```

Comment:
Is this logic that should get pulled out into its own method so that it can be reused?

Reply:
what do you mean? this is the method... i need more explanation please